x/pool/types/keys.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "cosmos-pool"
```

Comment:
should this just be pool instead?

Reply:
marko said that there were other modules that are called pool, this would conflict with those

Reply:
I see. I guess my disposition is that `cosmos-pool` isn't descriptive enough. But just my 2 cents. I would start with asking, "What is the purpose of this module?", "What/how is this 'pool' to be used?" Is it a community pool only? Then perhaps consider `community-pool` or `protocol-pool`

Reply:
community-pool makes sense